src/App.tsx

```diff
@@ -11,14 +11,17 @@ export class App extends React.Component<{}, State> {
   };
 
   addOne = () => {
-    // write code here
+    this.setState((state) => ({
```

Comment:
```suggestion
    this.setState((prevState) => ({
```

not a big mistake, but it is usually called `prevState`